phpstan.neon

```diff
@@ -16,7 +16,6 @@ parameters:
         - compiler/src
 
     excludes_analyse:
-        - "*/Expected/*"
```

Comment:
This change is just for demoing purposes (and verification)

Will/need to be reverted before merge